src/components/facts/list/container.js

```diff
@@ -0,0 +1,32 @@
+import React, {Component, PropTypes} from 'react'
+import {connect} from 'react-redux'
+
+import * as organizationActions from 'gModules/organizations/actions'
+
+import {FactList} from './list.js'
+
+function mapStateToProps(state) {
+  return {
+    organizations: state.organizations,
+    organizationFacts: state.facts.organizationFacts,
+  }
+}
+
+@connect(mapStateToProps)
+export class FactListContainer extends Component{
```

Comment:
This shouldn't have to re-select the organization like this. Instead of passing in the organization Id, pass the whole org and you can skip that selection.